### PR TITLE
remove unused css and also 'visited' styling

### PR DESCRIPTION
**Pull request by mattip**
xref hpyproject/hpyproject.org#15, doint it here instead since there are more pages to preview the change

### CODE REVIEW

File: css/site.css [hunk 1]
```diff
@@ -547,11 +509,6 @@ body > #body-outer {
 	padding: 10px 12px 10px 0;
 }
 
-.quote-attribution a, .quote-attribution a:active, .quote-attribution a:visited {
-	color: #575757;
-	text-decoration: underline;
-}
-
```
Comment (mattip) [inline css/site.css:554]: I think this is the real change that removes the styling from `a:visited`

**Comment by mattip** [comment]
@jablonskidev this site uses the same css as hpyproject.org. Can you checkout the styling of a few pages in the [preview](https://deploy-preview-25--keen-mestorf-442210.netlify.app/) and confirm it is better than the [web site](https://pypy.org)? Once is merged I will make a PR to hpyproject.org with these changes

**Comment by antocuni** [comment]
This is a screenshot of how it looks on my machine:
image

I think that the purple "PyPy" title looks a bit ugly. What about making the links always blue, without distinguishing between visited and unvisited? I think this is the approach taked by e.g. github itself

**Comment by cfbolz** [comment]
Fwiw, I dislike the recent trend towards not showing a difference for visited links (but feel free to ignore me)

**Comment by mattip** [comment]
> What about making the links always blue, without distinguishing between visited and unvisited?

Reading about [links and accessibility](https://webaim.org/blog/wcag-2-0-and-link-colors) that seems like it might be a good choice, as well as adding back "underline", which is removed by the base Nikola `theme.css`.  Done in latest commit.

**Comment by mattip** [comment]
@cfbolz can you suggest a color for visited links that looks good with the theme (which tried to maintain some consistency with the logo's greens, so purple clashes) and meets some accessibility requirements for discrimination from the other colors?

**Comment by mattip** [comment]
I am going to put this in to move it forward, but am happy to change it in the future as well.

**Pull request merged**